src/components/mixins/DownloadMixin.vue

```diff
@@ -0,0 +1,32 @@
+<script lang='ts'>
+import Vue from 'vue';
+import Component from 'vue-class-component';
+
+import ThunderstoreMod from "../../model/ThunderstoreMod";
+import Profile from "../../model/Profile";
+
+
+@Component
+export default class DownloadMixin extends Vue {
+    get ignoreCache(): boolean {
```

Comment:
This is currently used only in DownloadModModal which redefines the method. Seems it's not needed in the mixin.